components/financial_assistance/app/views/financial_assistance/incomes/_job_income.html.erb

```diff
@@ -1,77 +1,125 @@
-<div id="<%= dom_id income %>" class="income">
-  <div class="display-income">
-    <div class="focus_effect_module">
-      <div class="row row-form-wrapper no-buffer row-height">
-        <div class="col-md-11 no-pd form-group-lg no-pd class-fa-household">
-          <label class="static_label label-floatlabel">EMPLOYER NAME</label>
-          <span class="field_value floatlabel form-control active-floatlabel" id="name"><%= income.employer_name %></span>
-        </div>
-        <div class="col-md-1 no-pd form-group-lg class-fa-household fa-adjustment">
-          <a tabindex="0" onkeydown="handleButtonKeyDown(event, 'income-edit-<%= dom_id income %>')" id="income-edit-<%= dom_id income %>" class="income-edit edit-pr"><i class="fa fa-pencil fa-lg" aria-hidden="true"></i>
-            <span class="hide"><%=l10n("edit")%></span>
-          </a>
-          <a tabindex="0" onkeydown="handleButtonKeyDown(event, 'income-delete-<%= dom_id income %>')" id="income-delete-<%= dom_id income %>" class="income-delete"><i class="fa fa-trash-o fa-lg" aria-hidden="true"></i>
-            <span class="hide"><%=l10n("delete")%></span>
-          </a>
+<% if @bs4 %>
+  <div id="<%= dom_id income %>" class="income my-4">
+    <div class="display-income">
+      <div class="focus_effect_module">
+        <div class="d-flex col-sm income-form-row p-0 mb-2">
+          <div class="w-100 pr-3">
+            <label class="static_label label-floatlabel required" for="name-<%= income.employer_name %>"><%= l10n("faa.incomes.employer") %></label>
+            <input class="fake-disabled-input w-100" id="name-<%= income.employer_name %>" disabled="true" value="<%= income.employer_name %>">
+          </div>
+          <div class="w-100">
+            <label class="static_label label-floatlabel required" for="phone-<%= income.employer_phone %>-<%= income.id %>"><%= l10n("faa.incomes.employer_phone") %></label>
+            <input class="fake-disabled-input w-100" id="phone-<%= income.employer_phone %>-<%= income.id %>" disabled="true" value="<%= income.employer_phone %>">
```

Comment:
should probably add a mask on this phone input, I know QA has been pushing for that

Reply:
masking added for (000) 000-0000 format